Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/ICounterValue.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation
+{
+    using Microsoft.ApplicationInsights.DataContracts;
+
+    /// <summary>
+    /// Interface represents the counter value.
+    /// </summary>
+    public interface ICounterValue
```

Comment:
It might be a lot of abstraction when we try to define the name of the metric within the collector. The name of the metric can be computed by the module from the "PerformanceCounterData" and it is independent of the collector being used. Different modules can have a different naming conventions, and that can be addressed if we just return the value as opposed to telemetry item. Please let me know if this is OK.